src/common/common.h

```diff
@@ -222,6 +222,10 @@
 #ifdef ARM
 #define FB_CPU CpuArm
 #endif /* ARM */
+#ifdef ARM64
+#define DARWIN64
```

Comment:
It does not seems correct to use `DARWIN64` (same as used for __x86_64__) here.

Reply:
You need to use new constant here (DARWINARM64?) and adjust src/remote/remote_def.h, src/remote/protocol.h and src/remote/inet.cpp with arch_darwin_arm64.

Reply:
It would also need changes in src/common/classes/DbImplementation.cpp.

Ww would better wait for @AlexPeshkoff feedback :)

Reply:
Certainly, using different constants for different CPu like you suggest will for sure work. And I think that right now in order to have port better sooner it's right choice.

But longterm we should better understand why are different ARCHITECTUREs needed for different CPU under MacOS. Currently they are ORed in inet.cpp. Is it to avoid symmetric protocol between them? If yes, why is it not needed under linux with different CPU?

Reply:
Looks like currently there is no need separating Darwin's ARCHITECTURE per HW. I.e. using DARWIN64 is OK for arm64.